### PR TITLE
fix: missing prereq step for an OCP3 installation

### PR DESCRIPTION
**Pull request by MichalMaler**
Signed-off-by: Michal Maléř <mmaler@redhat.com>

> Read our [Contribution guide](https://github.com/eclipse/che-docs/blob/master/CONTRIBUTING.adoc) before submitting a PR.

### What does this PR do?

Adding missing prerequisite that is needed during a special event on OCP 3.11

### What issues does this PR fix or reference?

https://github.com/eclipse/che/issues/20004

### Specify the version of the product this PR applies to.
7.30.x and beyond

### PR Checklist

As the author of this Pull Request I made sure that:

- [ ] `vale` has been run successfully against the PR branch
- [ ] Link checker has been run successfully against the PR branch
- [ ] Documentation describes a scenario that is already covered by QE tests, otherwise an issue has been created and acknowledged by Che QE team
- [ ] Changed article references are updated where they are used (or a redirect has been configured on the docs side):
    - [ ] Dashboard [default branding data](https://github.com/eclipse-che/che-dashboard/blob/main/src/services/bootstrap/branding.constant.ts)
    - [ ] Chectl [constants.ts](https://github.com/che-incubator/chectl/blob/master/src/constants.ts)

### CODE REVIEW

File: modules/installation-guide/partials/proc_installing-che-on-openshift-3-using-the-operator.adoc [hunk 1]
```diff
@@ -36,6 +36,8 @@ This approach is only supported for use with {ocp} and {osd} version 3.11, but a
 
 * __<namespace>__ represents the project of the target installation.
 
+* Configure OpenShift to pull images from `registry.redhat.com`. See link:https://access.redhat.com/RegistryAuthentication[Red Hat Container Registry Authentication]
```
Comment (rkratky) [inline modules/installation-guide/partials/proc_installing-che-on-openshift-3-using-the-operator.adoc]:
```suggestion
* Configure OpenShift to pull images from `registry.redhat.com`. See link:https://access.redhat.com/RegistryAuthentication[Red Hat Container Registry Authentication].
```

**Review by tolusha — APPROVED** [review]

**Comment by themr0c** [comment]
For 7.32.x only?

**Comment by dmytro-ndp** [comment]
@MichalMaler: please, make sure that PR satisfies requirements expressed in `PR Checklist`.

**Comment by MichalMaler** [comment]
@dmytro-ndp Sorry sir. Here is the JIRA reference https://issues.redhat.com/browse/RHDEVDOCS-2972
Milestone, project, prefix, and reason were already set. What you were missing?

**Review by themr0c — APPROVED** [review]

**Pull request merged**